src/fetch_data.py

```diff
@@ -0,0 +1,36 @@
+import requests
+import os
+from src.logs.logger import DataProcessingLogger
+from src.services.IndexerService import IndexerService
+
+
+
+def main():
+    service = IndexerService()
+
+    query = {
+        "operationName": "getTransactions",
+        "variables": None,
+        "query": """query getTransactions {
+                        nftTransferEntities(limit: 100) {
+                            amount
+                            id
+                            fromAddress
+                            toAddress
+                            nft {
+                            id
+                            }
+                        }
+                    }"""
+        }
+
+    headers = {
+         "Content-Type": "application/json",
+            "Accept": "application/json"
+    }
```

Comment:
I don't think headers would change so we can put those also in the runQuery method withou passing them